src/bin/customer/establish.rs

```diff
@@ -613,6 +535,247 @@ fn write_establish_json(establishment: &Establishment) -> Result<(), anyhow::Err
     Ok(())
 }
 
+/// Parse the [`Amount`]s passed on the command line into the `Balance` types used throughout.
+fn parse_initial_balances(
```

Comment:
Easy to unit test

Reply:
Good call, added a test with the failure cases I know about.